### PR TITLE
Cleanup readme file and copy build instructions from lucene subfolder…

### PR DESCRIPTION
**Pull request by uschindler**
…. Also update doap files, remove script clearly used for solr (more to come later)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,25 +31,63 @@ comprehensive documentation, visit:
 
 ## Building with Gradle
 
-### Building Lucene
+### Basic steps:
+  
+  0. Install OpenJDK 11 (or greater)
+  1. Download Lucene/Solr from Apache and unpack it
```
Comment (janhoy) [inline README.md:37]: Download lucene... (well, we don't yet have a release with only lucene...)

Also on line 30  -don't need Lucene: as prefix

File: README.md [hunk 2]
```diff
@@ -31,25 +31,63 @@ comprehensive documentation, visit:
...
 don't, then we suggest starting at https://www.oracle.com/java/ and learning
-more about Java, before returning to this README. Solr runs with
+more about Java, before returning to this README. Lucene runs with
 Java 11 and later.
 
-As of 9.0, Lucene/Solr uses [Gradle](https://gradle.org/) as the build
-system. Ant build support has been removed.
+Lucene uses [Gradle](https://gradle.org/) for build control.
+
+NOTE: When Solr moves to a Top Level Project, it will no longer
```
Comment (janhoy) [inline README.md:50]: "When Solr moves to a Top Level Project" - has happened :) 
Is it necessary to download Solr to build lucene now?

File: README.md [hunk 3]
```diff
@@ -31,69 +31,83 @@ comprehensive documentation, visit:
...
-Note that the gradle build does not create or copy binaries throughout the
-source repository so you need to switch to the packaging output folder above;
-the rest of the instructions below remain identical. The packaging directory 
-is rewritten on each build. 
+### Step 2) Change directory (cd) into the top-level directory of the source tree
 
-For development, especially when you have created test indexes etc, use
-the `./gradlew dev` task which will copy binaries to `./solr/packaging/build/dev`
-but _only_ overwrite the binaries which will preserve your test setup.
+The parent directory for both Lucene and Solr contains the base configuration
```
Comment (janhoy) [inline README.md:75]: Solr is removed from this repo...

File: README.md [hunk 4]
```diff
@@ -31,69 +31,83 @@ comprehensive documentation, visit:
...
+
+NOTE: DO NOT use `gradle` command that is already installed on your machine (unless you know what you'll do).
+The "gradle wrapper" (gradlew) does the job - downloads the correct version of it, setups necessary configurations.
+
+The first time you run Gradle, it will create a file "gradle.properties" that
+contains machine-specific settings. Normally you can use this file as-is, but it
+can be modified if necessary.
 
-`bin/solr start`
+`./gradlew check` will assemble Lucene/Solr and run all validation
```
Comment (janhoy) [inline README.md:98]: remove solr

**Review by janhoy — CHANGES_REQUESTED** [review]
Need to get rid of lots of solr references. Lucene users won't need to download Solr to build lucene, will they?

File: README.md (hunk 4 above)
Comment (uschindler) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (uschindler) [reply README.md]: done.

File: README.md (hunk 2 above)
Comment (uschindler) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (uschindler) [reply README.md]: done

**Comment by uschindler** [comment]
I just copypasted the BUILD.md from lucene, but that was outdated, too.

No more occurences of Solr in the Toplevel README

**Comment by uschindler** [comment]
I also merged in changes from @rmuir PR #5: The PR template was also adapted

**Review by janhoy — APPROVED** [review]

**Review by rmuir — APPROVED** [review]

File: .github/workflows/hunspell.yml [hunk 5]
```diff
@@ -3,7 +3,7 @@ name: Hunspell regression tests
 on:
   pull_request:
     branches:
-      - 'master'
+      - 'main'
```
[the remark this answers is not in the document]
Comment (rmuir) [reply .github/workflows/hunspell.yml:6]: no worries, trust me there is more to fix. i just ran `git grep master` and tried to deal with the simplest of them.

[the remark this answers is not in the document]
Comment (dweiss) [reply .github/workflows/hunspell.yml:6]: I know. And I appreciate. I think it should be downhill from now on though - Lucene alone is a much smaller codebase to work with (and very focused compared to Solr's ecosystem). Step by step. I'll go through the build files in a spare moment too.

**Pull request merged**